test/Telegram.Bot.Tests.Integ/Polls/PollMessageTests.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Telegram.Bot.Requests;
+using Telegram.Bot.Tests.Integ.Framework;
+using Telegram.Bot.Types;
+using Telegram.Bot.Types.Enums;
+using Xunit;
+
+namespace Telegram.Bot.Tests.Integ.Polls
+{
+    [Collection(Constants.TestCollections.NativePolls)]
+    [TestCaseOrderer(Constants.TestCaseOrderer, Constants.AssemblyName)]
+    public class PollMessageTests : IClassFixture<PollTestsFixture>
+    {
+        private ITelegramBotClient BotClient => _fixture.BotClient;
+
+        private readonly PollTestsFixture _classFixture;
+        private readonly TestsFixture _fixture;
+
+        public PollMessageTests(TestsFixture fixture, PollTestsFixture classFixture)
+        {
+            _fixture = fixture;
+            _classFixture = classFixture;
+        }
+
+        [OrderedFact(FactTitles.ShouldSendPoll)]
+        [Trait(Constants.MethodTraitName, Constants.TelegramBotApiMethods.SendPoll)]
+        public async Task Should_Send_Poll()
+        {
+            string pollQuestion = "Poll question";
+            string[] pollOptions =
+            {
+                "Poll option 1",
+                "Poll option 2",
+            };
```

Comment:
We are using a more verbose and explicit style of code for the test cases to make it even easier to read and understand to new users.

e.g. https://github.com/TelegramBots/Telegram.Bot/blob/master/test/Telegram.Bot.Tests.Integ/Sending%20Messages/AnimationMessageTests.cs#L26